### DISCUSSION TITLE
性格「しあわせもの」を底上げして欲しい。

### DISCUSSION THREAD

**Discussion by ironside777** [hengband/hengband#618] [Ideas]
性格「しあわせもの」は勝利者が突出して少なく、ベーシックな性格の中では人気が無いので、
てこ入れして欲しい。
焼け石に水かもしれませんが、プリースト寄りの性格ということを考慮して、
・魔法・超能力・ものまねの難度-2
・魔法道具支配+2→+4
・魔法防御+4→+5
などいかがでしょうか。

**Comment by iks3** [comment]
ODSNのチケットにあったものを転載。今後も考える。
調整アイデア例：
ふつう: ふつうじゃない何かの能力に目覚める(ランダムで)
ちからじまん: ACにボーナス、腕力が上がりやすい
きれもの: 現ボーナスに加え、第2領域も達人まで熟練度が上がる
しあわせもの: 呪われにくくなる、呪いの効果が発動しにくくなる
すばしっこい: 加速とは別に、たまに行動時の消費エネルギーが少し減る
いのちしらず: 恐怖耐性、パラノイアの変異が付かない
がまんづよい: 被ダメージが微妙に減る(死にそうなとき？)

**Reply by deskull-m** [reply]
近日Issue化してみますかねえ。
